.github/workflows/version_release.yml

```
@@ -0,0 +1,32 @@
+
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run semantic release
+        run: npx semantic-release
+        working-directory: ${{ github.workspace }}
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)